client/src/pages/submitSecret/form.js

```diff
@@ -125,12 +163,14 @@ const Form = ({formResponse}) => {
           <input id="files" type="file" onChange={handleFileChange} className="form-control"/>
           <div id="files-list" className="row row-cols-4 g-1 mt-1"></div>
           <div className="col">
-            <div className="row">
-              <label for="passphrase" className="col-sm-2 col-form-label">Secret key</label>
-              <div className="col-sm-10">
+              <label id="div-input-pass-pub-label" for="passphrase" className="col-sm-2 col-form-label">Secret key</label>
+              <div id="div-input-pass-pub" className="col-sm-10">
                 <input type="password" id="passphrase" name="passphrase" onChange={handleInputChange} className="form-control"/>
               </div>
-            </div>
+              <div className='row'>
```

Comment:
Put the label and inputs back inside the `<div className="row">`

Reply:
Fixed in 14d810a